src/Illuminate/Filesystem/Filesystem.php

```diff
@@ -352,7 +352,11 @@ public function copy($path, $target)
     public function link($target, $link)
     {
         if (! windows_os()) {
-            return symlink($target, $link);
+            if (function_exists('symlink')) {
+                return symlink($target, $link);
+            }else{
```

Comment:
I guess, you are missing an additional space on both sides for the style check to pass:
```suggestion
            } else {
```